apispec/openapi.json

```diff
@@ -5201,32 +5201,32 @@
       "transport.id": {
         "title": "transportId",
         "type": "integer",
-        "description": "Indicates an transport id, it is a unique integer number in HDT System."
+        "description": "Indicates a transport id. It is a unique integer number in the HDT system."
       },
       "transport.code": {
         "title": "transportCode",
         "type": "string",
-        "description": "if transport type of this transport is ipport, it uses this form ‘targetDomain:targetPort’ to represent. if transport type of this transport is vpn or socks5, it uses ‘userAccount’ to represent."
+        "description": "If transport type of this transport is ipport, it uses this form ‘targetDomain:targetPort’ to represent. If transport type of this transport is vpn or socks5, it uses ‘userAccount’ to represent."
```

Comment:
Suggestion:   "If the transport type is ipport", the transport code should be in this format 'targetDomain:targetPort’'.  If the transport type is vpn or socks5, the transport cdeoe should be a userAccount."

(I don't know what a user account is, though."